### DISCUSSION TITLE
FutureWarning from Pandas

### DISCUSSION THREAD

**Discussion by hbc-svolkmer** [meteostat/meteostat-python#156] [General]
Just got a FutureWarning while using meteostat.Hourly: 

_FutureWarning: 'H' is deprecated and will be removed in a future version, please use 'h' instead._

This is probably caused by the usage of pandas.to_timedelta()
It would be great if this could be taken into account in the next version. 
Many thanks to the developers.

**Comment by clampr** [comment]
Thank you @hbc-svolkmer. We will consider it for the next version.
